autodo/models.py

```
@@ -99,3 +99,292 @@ class Refueling(models.Model):
+        null=True,
+    )
+    created_by = models.ForeignKey(
+        User,
+        related_name="created_work_order",
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+    )
+
+    class Meta:
+        ordering = ["-opened_at"]
```

Ruff (0.15.0)

[warning] Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)